src/main/java/dev/callumherr/modding/ducky_lib/utils/Instances.java

```diff
@@ -0,0 +1,19 @@
+package dev.callumherr.modding.ducky_lib.utils;
+
+public class Instances
```

Comment:
Dont quite understand why this is needed

Reply:
just for temp purpose it was